examples/fake-training.py

```diff
@@ -74,10 +74,12 @@ def generate_accuracy_curve(epoch, max_epochs, max_acc=0.95, min_acc=0.1):
 
         wandb.log(
             {
-                "train_loss": round(train_loss, 4),
-                "train_accuracy": round(train_accuracy, 4),
-                "val_loss": round(val_loss, 4),
-                "val_accuracy": round(val_accuracy, 4),
+                "train/loss": round(train_loss, 4),
+                "train/accuracy": round(train_accuracy, 4),
+                "train/random/seed1": random.random(),
+                "train/random/seed2": random.random(),
```

Comment:
It's a bit strange to log the seed, I'd suggest logging rewards, which is more aligned with what you could find with multi-objective training

```suggestion
                "train/rewards/reward1": random.random(),
                "train/rewards/reward2": random.random(),
```

Reply:
hehe thanks I agree